### DISCUSSION TITLE
Azure GDPR raport

### DISCUSSION THREAD

**Discussion by mutaverat** [AzureCommunityPL/group#477] [Pomoc]
Witam
Mam takie zadanie aby zrobić raport GDPR tego co mam w Azure. 
W security są dostępne rózne raport od SOC az do PCI ale nie GDPR.
Czy znacie jakieś rozwiązanie aby zrobić taki raport. Moze jakieś narzędzia tylko nie firm trzecich. 
Lub czy ktoś coś podobnego robił?

**Comment by mgrabarz** [comment]
Co taki raport ma zawierać?

**Reply by mutaverat** [reply]
Wystarczy aby mi sprawdził bazę danych i pokazał czy mam tam dane które są nie zgodne z GDPR np dane o numerach dowodów.

**Reply by mgrabarz** [reply]
Obawiam się, że zawartość bazy danych to nie pytanie do "platformy" Azure, a do "aplikacji" która w tej bazie trzyma dane. Ja bym od razu leciał do ownera bazy i poprosił od niego/niej taki raport.

Przecież toole Azure nie zaglądają do zawartości bazy, a już na pewno technicznie nie byłoby to proste, by umiały analizować zawartość pod kątem jakiejkolwiek zgodności. To co możesz uzyskać w kontekście GDPR to co najwyżej rezydencja danych.

**Reply by mgrabarz** [reply]
A no i inna kwesta co znaczy "zgodne". Trzymanie numerów dowodów jest zgodne, czy nie jest? Osobiście uważam, że może być zgodne, jeśli zebraliście odpowiednie zgody w aplikacji, polityka prywatności ten temat odpowiednio ujęła itp. Skąd ten biedny Azure ma wiedzieć, czy aplikacja ogarnęła kuwetę zgodnie z lokalnym prawem :D

**Reply by mutaverat** [reply]
A czy mogę sprawdząc storage i czy tam coś nie lezy co moze zawierać dane? CZy są do tego narzędzia? 
Bo z bazą to masz rację i moze być trudno ale co z storage i tym co tam moze się znajdować?

**Reply by pawelmitrus** [reply]
Dokładnie tak samo jak z bazą - jesteś trochę więcej w stanie zrobić z poziomu platformy Azure (o ile masz ofc odpowiednie uprawnienia, bo dostępu do danych wewnątrz Storage nie dostaniesz mając np. kontrybutora na samym zasobie), ale musisz mieć wiedzę CO jest na tym storage, żeby być w stanie to jakoś sensownie przeszukać.

**Reply by smereczynski** [reply]
Raczej chodzi o klasyfikację danych, czyli Purview. [LINK](https://learn.microsoft.com/en-us/azure/purview/concept-classification?WT.mc_id=AZ-MVP-5001618)

**Comment by mgajownik** [comment]
cześć zobacz czy nie szukasz AIP
https://learn.microsoft.com/en-us/azure/information-protection/what-is-information-protection
